### PR TITLE
Bug 1106175 - Align conference group UI participants to right for RTL. r...

### PR DESCRIPTION
**Pull request by tamarahills**
...=gsvelto.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=731e3f2a977e)

File: apps/callscreen/style/conference_group_ui.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+html[dir="rtl"] #group-call-details-list > section .additionalContactInfo {
+  text-align: right;
+  padding: 0 0 2rem 0;
+}
+
```
Comment (gabrielesvelto) [inline apps/callscreen/style/conference_group_ui.css]: nit: Please move this below the regular LTR selector; I think it's clearer to have the RTL rule below each corresponding LTR selector

File: apps/callscreen/style/conference_group_ui.css [hunk 2]
```diff
@@ -39,6 +44,9 @@
   bottom: 0;
   left: 0;
 }
+html[dir="rtl"] #group-call-details .hangup-button {
+  background: url('images/handled_call/actionicon_activecall_hangup.png') no-repeat scroll left 1.8rem center / 4rem auto transparent;
+ }
```
Comment (gabrielesvelto) [inline apps/callscreen/style/conference_group_ui.css]: nit: The closing brace has a space before it, also please leave an empty line between the previous ruleset and this one

File: apps/callscreen/style/conference_group_ui.css [hunk 3]
```diff
@@ -51,6 +59,11 @@
   right: 1.5rem;
 }
 
+html[dir="rtl"] #group-call-details section.ended > .duration {
+ right: auto;
+ left: 1.5rem;
```
Comment (gabrielesvelto) [inline apps/callscreen/style/conference_group_ui.css]: nit: Indentation should be two spaces, it's only one here

File: apps/callscreen/style/conference_group_ui.css [hunk 4]
```diff
@@ -61,6 +74,10 @@
   line-height: 1.7rem;
 }
 
+html[dir="rtl"] #group-call-details section.ended > .duration > .total-duration {
+ text-align: left;
```
Comment (gabrielesvelto) [inline apps/callscreen/style/conference_group_ui.css]: nit: Likewise

**Pull request merged**